docs/src/how-to/vscode.md

```diff
@@ -18,14 +18,14 @@ environment.systemPackages = [
 [nix-ld](https://github.com/Mic92/nix-ld) is a program that provides `/lib64/ld-linux-x86-64.so.2`,
 allowing foreign binaries to run on NixOS.
 
-Running the VSCode server on NixOS-WSL requires using [nix-ld-rs](https://github.com/nix-community/nix-ld-rs), instead of the regular nix-ld.
+Running the VSCode server on NixOS-WSL requires using nix-ld 2.0 which is as often writing only on unstable or [nix-ld-rs](https://github.com/nix-community/nix-ld-rs).
```

Comment:
I think something went wrong with this sentence. At least my brain refuses to parse it

Reply:
```suggestion
Running the VSCode server on NixOS-WSL requires using nix-ld 2.0 which is as of writing only on NixOS unstable or [nix-ld-rs](https://github.com/nix-community/nix-ld-rs) on NixOS 24.05.
```

Reply:
derp